packages/playback-core/src/index.ts

```diff
@@ -514,26 +516,7 @@ export const setupMux = (
   }
 };
 
-export const loadMedia = (
-  props: Partial<Pick<MuxMediaProps, 'preferPlayback' | 'src' | 'type' | 'startTime' | 'streamType' | 'autoplay'>>,
-  mediaEl: HTMLMediaElement,
-  hls?: Pick<
-    Hls,
-    | 'config'
-    | 'on'
-    | 'once'
-    | 'startLoad'
-    | 'stopLoad'
-    | 'recoverMediaError'
-    | 'destroy'
-    | 'loadSource'
-    | 'attachMedia'
-    | 'liveSyncPosition'
-    | 'subtitleTracks'
-    | 'subtitleTrack'
-    | 'userConfig'
-  >
-) => {
+export const loadMedia = (props: Partial<MuxMediaProps>, mediaEl: HTMLMediaElement, hls?: HlsInterface) => {
```

Comment:
I'm cool with simplifying these type defs, though the original intent was to make sure we were being intentional about our API dependencies.